draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -768,24 +763,26 @@ guess. A new salt MUST be chosen for each claim.
 
 The RECOMMENDED minimum length of the randomly-generated portion of the salt is 128 bits.
 
-Note that minimum 128 bits would be necessary when SHA-256, HMAC-SHA256, or a function of similar strength is used, but a smaller salt size might achieve similar level of security if a stronger iterative derivation function is used.
+Note that minimum 128 bits would be necessary when SHA-256, or a function of similar strength is used.
```

Comment:
I don't think this sentence adds information, the RECOMMENDATION is already there.
```suggestion
```